common/src/thrift/tachyon.thrift

```diff
@@ -220,6 +220,9 @@ service FileSystemMasterService {
   void setPinned(1: i64 fileId, 2: bool pinned)
     throws (1: TachyonTException e)
 
+  void setTTL(1: i64 fileId, 2: i64 ttl)
```

Comment:
The SetState RPC would take a single parameter: `SetStateOptions` this is a struct in which all fields are optional and Thrift generates methods that can be used to check whether the fields have been set. Also, that way we can add more options without breaking backwards compatibility.